main.ts

```diff
@@ -208,18 +207,11 @@ class Vector2D extends Entity {
 class Line extends Entity {
     public startPosition: Vector2D;
     public endPosition: Vector2D;
-    public collision_direction: Vector2D;
```

Comment:
I like this change. A line should be as simple a construct as two points, and it shouldn't know what ground/tire is.